src/Artsy/Relay/createEnvironment.ts

```diff
@@ -118,3 +122,25 @@ export function createEnvironment(config: Config = {}) {
 
   return environment
 }
+
+/**
+ * During the client-side rehydration phase take SSR cache and add to Relay's
+ * QueryResponseCache, which is used inside of cacheMiddleware.
+ *
+ * @param cache RelayQueryResponseCache
```

Comment:
Couldn't find a type for this

Reply:
Care to send a DT PR?

Reply:
Curious how one should approach types for things that are buried like this: 

```
import RelayQueryResponseCache from 'relay-runtime/lib/RelayQueryResponseCache';
```

Would it just sit in the top-level `relay-runtime` package?

Reply:
Ah no, actually you just rebuild the file tree. See e.g. https://github.com/DefinitelyTyped/DefinitelyTyped/tree/master/types/graphql